edge-apps/asset-metadata/static/js/main.js

```diff
@@ -1,130 +1,181 @@
-    strokeWeight: 0,
-    fillColor: '#C4C4C4',
-    fillOpacity: 0.30,
-    map,
-    center: new google.maps.LatLng(lat, lng),
-    radius: 50000
-  })
+/* global screenly, Sentry */
+/* eslint-disable-next-line no-unused-vars, no-useless-catch */
 
-  new google.maps.Circle({
-    strokeWeight: 0,
-    fillColor: '#C4C4C4',
-    fillOpacity: 0.20,
-    map,
-    center: new google.maps.LatLng(lat, lng),
-    radius: 80000
+const sentryDsn = screenly.settings.sentry_dsn
+// Initiate Sentry.
+if (sentryDsn) {
+  Sentry.init({
+    dsn: sentryDsn
   })
+} else {
+  console.warn('Sentry DSN is not defined. Sentry will not be initialized.')
 }
 
-function initApp() {
-  function setValue(elClass, value = 'Not provided') {
-    document.querySelector(elClass).innerText = value
-  }
+document.addEventListener('DOMContentLoaded', async () => {
+  async function initApp () {
+    const {
+      coordinates,
+      hostname,
+      hardware,
+      screenly_version: screenlyVersion,
+      screen_name: screenName,
+      tags
+    } = screenly.metadata
 
-  function loadGoogleMaps() {
-    const script = document.createElement('script')
-    const googleMapsApiKey = screenly.settings.google_maps_api_key
-    const queryParams = [
-      'callback=initMap',
-      `key=${googleMapsApiKey}`
-    ].join('&')
-    script.src = `https://maps.googleapis.com/maps/api/js?${queryParams}`
-    script.async = true
-    document.body.appendChild(script)
-  }
+    document.querySelector('.host-name-text').innerText = hostname
+    document.querySelector('.screen-name-text').innerText = screenName
+    document.querySelector('.hardware-name-text').innerText = hardware
+    document.querySelector('.version-name-text').innerText = screenlyVersion
+    document.querySelector('.coordinates-name-text').innerText = formatCoordinates(coordinates)
 
-  function formatCoordinates(coordinates) {
-    const [lat, lng] = coordinates;
-    const formatQuantity = (coordinate, type) => {
-      return `${Math.abs(coordinate).toFixed(4)}\u00B0`
+    // Raw Coordinates converting to formatted coordinates
+    function formatCoordinates (coordinates) {
+      const [lat, lng] = coordinates
+      const latString = `${Math.abs(lat).toFixed(4)}\u00B0`
+      const latDirection = lat > 0 ? 'N' : 'S'
+      const lngString = `${Math.abs(lng).toFixed(4)}\u00B0`
+      const lngDirection = lng > 0 ? 'E' : 'W'
+
+      return `${latString} ${latDirection}, ${lngString} ${lngDirection}`
     }
-    const formatLatitude = (lat) => {
-      const direction = lat > 0 ? 'N' : 'S'
-      return `${formatQuantity(lat)} ${direction}`
+
+    // Labels
+    const labelChipContainer = document.querySelector('.label-chip-container')
+    tags.forEach(label => {
+      const chip = document.createElement('div')
+      chip.classList.add('label-chip')
+      chip.innerHTML = `<span class="label-chip-text-label">${label}</span>`
+      labelChipContainer.appendChild(chip)
+    })
```

Comment:
Prefer using document.createDocumentFragment here. Append the chips to teh fragment and finally to labeChipContainer

Reply:
@iamplugged updated 

```
 // Labels
    const labelChipContainer = document.querySelector('.label-chip-container')
    const fragment = document.createDocumentFragment()
    tags.forEach(label => {
      const chip = document.createElement('div')
      chip.classList.add('label-chip')
      chip.innerHTML = `<span class="label-chip-text-label">${label}</span>`
      fragment.appendChild(chip)
    })
    labelChipContainer.appendChild(fragment)
```